homeassistant/components/androidtv/entity.py

```diff
@@ -85,13 +85,20 @@ async def _adb_exception_catcher(
                 # pylint: disable-next=protected-access
                 self._attr_available = False
                 return None
-            except Exception:
+            except Exception as err:  # pylint: disable=broad-except
                 # An unforeseen exception occurred. Close the ADB connection so that
-                # it doesn't happen over and over again, then raise the exception.
+                # it doesn't happen over and over again.
+                _LOGGER.error(
```

Comment:
Is this going to log over and over?

https://developers.home-assistant.io/docs/integration_quality_scale_index?_highlight=quality#silver-

> Handles internet unavailable. Log a warning once when unavailable, log once when reconnected.
> Handles device/service unavailable. Log a warning once when unavailable, log once when reconnected.

Reply:
I added a check if available before logging, this should avoid to repeat error log.